### ISSUE TITLE
Big thank you and a suggestion

### ISSUE THREAD

**Issue by sectune** [flabbamann/aha_region#32]
First of all, thank you so much for this! Please publish a way to express my gratitude as a small donation to show my appreciation!

Small suggestion: as HA moves towards fully in-GUI config, I was wondering if a method to implement an in-GUI config to enter the needed data now located in config.yaml is on your roadmap?

**Comment by flabbamann** [comment]
Yes, I want that too. And I would like to interactively load the address data during setup so no one has to dig through aha website sources anymore 😉.

But I don't have much time currently, so don't expect anything soon... 😐

**Comment by lmaertin** [comment]
I took this, please have an eye on my pull request.

**Comment by lmaertin** [comment]
<img width="575" height="229" alt="Image" src="https://github.com/user-attachments/assets/1d6cabfd-628e-4599-bfe1-6576c4795e09" />

<img width="576" height="242" alt="Image" src="https://github.com/user-attachments/assets/c20d0138-091d-440e-9582-4b0bfa0f38ad" />

<img width="581" height="553" alt="Image" src="https://github.com/user-attachments/assets/7f2318ef-ca9d-4fe7-b303-37cb0da72ff9" />


...

**Comment by lmaertin** [comment]
yaml config still works, for compatibility reasons and if AHA ever changes the website layout and the parser fails...
